e2e/tests/121-Portal/MakeNewPayment/OCW/SendPaymentInstructionsOCW.spec.ts

```diff
@@ -0,0 +1,116 @@
+import HomePage from '@121-e2e/pages/Home/HomePage';
+import LoginPage from '@121-e2e/pages/Login/LoginPage';
+import NavigationModule from '@121-e2e/pages/Navigation/NavigationModule';
+import PaymentsPage from '@121-e2e/pages/Payments/PaymentsPage';
+import RegistrationDetails from '@121-e2e/pages/RegistrationDetails/RegistrationDetailsPage';
+import TableModule from '@121-e2e/pages/Table/TableModule';
+import { SeedScript } from '@121-service/src/scripts/seed-script.enum';
+import NLRCProgram from '@121-service/src/seed-data/program/program-nlrc-ocw.json';
+import { seedIncludedRegistrations } from '@121-service/test/helpers/registration.helper';
+import {
+  getAccessToken,
+  resetDB,
+} from '@121-service/test/helpers/utility.helper';
+import { registrationsOCW } from '@121-service/test/registrations/pagination/pagination-data';
+import { test } from '@playwright/test';
+import { AppRoutes } from '../../../../../interfaces/Portal/src/app/app-routes.enum';
+import englishTranslations from '../../../../../interfaces/Portal/src/assets/i18n/en.json';
+
+const nlrcOcwProgrammeTitle = NLRCProgram.titlePortal.en;
+const paymentLabel = englishTranslations.page.program.tab.payment.label;
+const paymentStatus = englishTranslations.entity.payment.status.success;
+const paymentFilterByMessage =
+  englishTranslations.entity.message['content-type']['generic-templated'];
+const messageContext =
+  englishTranslations.entity.message['content-type'].payment;
+const messageType = englishTranslations.entity.message.type.whatsapp;
+
+const cancel = englishTranslations.common.cancel;
+const status = 'success';
```

Comment:
Does it have translation?

Reply:
I did not find it in en.json. The status for 'success' under payment is 'Successful'. This status 'success' is stored in Excel file .

Reply:
Got it